internal/controllers/cluster/cluster_controller.go

```diff
@@ -67,6 +68,8 @@ type Reconciler struct {
 	Client    client.Client
 	APIReader client.Reader
 
+	RuntimeClient runtimeclient.Client
```

Comment:
let's drop the client again from this cluster controller as we don't use it (same for alias.go / main.go)